taxi/views.py

```diff
@@ -34,6 +35,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_create.html"
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_create.html"
```

Comment:
The `template_name` for `ManufacturerUpdateView` is set to `manufacturer_create.html`. Consider using a more appropriate template name like `manufacturer_update.html` to avoid confusion and ensure clarity.